app/scripts/modules/netflix/pagerDuty/pagerDuty.read.service.ts

```diff
@@ -0,0 +1,20 @@
+import {module} from 'angular';
+import {API_SERVICE} from "../../core/api/api.service";
+
+export class PagerDutyReader {
+  static $inject = ['API'];
+
+  constructor(private API: any) {}
+
+  public listServices(): ng.IPromise<any> {
+    return this.API.one('pagerDuty/services').useCache().get();
```

Comment:
change `.get()` to `.list()`, as this returns a collection (not 100% sure about this one - test it out and verify it works)